docs/source/release_notes/51_0.rst

```diff
@@ -42,7 +42,7 @@ Users/Test Writers
   ``--execution-order=variants-per-test``.
 
 * Test methods on parent classes are now found upon the use of the new
-  :ref:`recursive <docstring-directive-recursive>` docstring
+  `recursive <docstring-directive-recursive>` docstring
```

Comment:
I'm usually against touching older release notes, but this would prevents broken links without changing the content, so looks good.

Reply:
I have considered replacing this with link to older version of readthedocs, but again, that might bring issues when we move to a different provider...